src/Catel.MVVM/Collections/FastObservableDictionary.cs

```diff
@@ -68,6 +68,8 @@ public FastObservableDictionary(int capacity, IEqualityComparer<TKey> comparer)
         }
         public FastObservableDictionary(IDictionary<TKey, TValue> dictionary, IEqualityComparer<TKey> comparer)
         {
+            Argument.IsNotNull(() => dictionary);
```

Comment:
In Catel we must never use the expression ones (just in case Catel.Fody doesn't do what it has to do), always use nameof(dictionary), dictionary instead.